plugin/v3/plugin.proto

```diff
@@ -190,6 +192,17 @@ message Write {
   message Response {}
 }
 
+message Transform {
+  message Request {
+    // marshalled arrow.Record
+    bytes record = 1;
+  }
+  message Response {
+    // marshalled arrow.Record
```

Comment:
As mentioned in the meeting, this is compatible with sending something other than `arrow.Record` in the future